### PR TITLE
Feature/add tv os crashes

### PR DESCRIPTION
**Pull request by jaelim-ms**
In this PR
1. PLCrashReporter for tvOS has been added to Mobile Center.
2. MobileCenterCrashes for tvOS has been added to Mobile Center.
3. SasquatchTV application now imports MobileCenterCrashes.
4. The test application has not been verified. This will be a separate task.

### CODE REVIEW

File: MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm [hunk 1]
```diff
@@ -242,7 +244,9 @@ - (instancetype)init {
     _logBufferDir = [MSCrashesUtil logBufferDir];
     _analyzerInProgressFile = [_crashesDir URLByAppendingPathComponent:kMSAnalyzerFilename];
     _didCrashInLastSession = NO;
+#if !TARGET_OS_TV
     _enableMachExceptionHandler = YES;
+#endif
```
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm]: I would suggest
```
#if TARGET_OS_TV
     _enableMachExceptionHandler = NO;
#else
     _enableMachExceptionHandler = YES;
#endif
```
So you don't need the macro line 525.

File: MobileCenter.xcworkspace/xcshareddata/xcschemes/AllDoc+FrameworksTVOS.xcscheme [hunk 2]
```diff
@@ -47,9 +61,9 @@
       <MacroExpansion>
          <BuildableReference
             BuildableIdentifier = "primary"
-            BlueprintIdentifier = "043121491EE0BFC7007054C5"
-            BuildableName = "MobileCenterTVOSFramework"
```
Comment (clpolet) [inline MobileCenter.xcworkspace/xcshareddata/xcschemes/AllDoc+FrameworksTVOS.xcscheme]: Shouldn't this scheme also generate the framework?

Comment (jaelim-ms) [inline MobileCenter.xcworkspace/xcshareddata/xcschemes/AllDoc+FrameworksTVOS.xcscheme]: This is a doc framework and it should generate tvOS framework as well.

Comment (clpolet) [inline MobileCenter.xcworkspace/xcshareddata/xcschemes/AllDoc+FrameworksTVOS.xcscheme]: Right :)

File: MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm (hunk 1 above)
Comment (jaelim-ms) [inline MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm]: That was my first implementation but we cannot eliminate the macro at line 525.
`PLCrashReporterSignalHandlerTypeMach` is not available on tvOS SDK so anyway we have to do this one.

**Comment by jaelim-ms** [comment]
TARGET_OS_TV is set to 1 on tvOS SDK for device and simulator. Must be like this.

File: MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm (hunk 1 above)
Comment (clpolet) [inline MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm]: OK

**Pull request merged**